### ISSUE TITLE
Make the backlinks from showcases to adaptation options visible

### ISSUE THREAD

**Issue by DenoBeno** [clarity-h2020/csis#76]
Adaptation options should link to showcases that use these adaptation options. E.g. in https://csis.myclimateservice.eu/adaptation-options-overview table.

**Comment by DenoBeno** [comment]
@DanielRodera : I understand that you are drupal developer. How much time do you have for working on CSIS features?

**Comment by patrickkaleta** [comment]
I took care of those backlinks. I created an additional [block display](https://csis.myclimateservice.eu/admin/structure/views/view/adaptation_options_overview/edit/block_1) in the Adaptation Options View. It takes the ID of the adaptation option and filters out all showcases referencing this ID in their `field_adaptation_options` field. 
That way we can get these backlinks in other places as well. Right now those backlinks are displayed in the [Adaptation options overview](https://csis.myclimateservice.eu/adaptation-options-overview):

![backlink from adapt opt to showcase](https://user-images.githubusercontent.com/42167036/63011618-f793d080-be88-11e9-87a8-4228c2e5a51c.png)
